src/routes/files.ts

```diff
@@ -35,9 +25,9 @@ export default async function files(fastify: FastifyInstance) {
         place.name,
         place.contact.properties.name,
         place.contact.properties.phone,
+        place.userRoles.join(' '),
```

Comment:
just reordering this. i personally prefer having username/password at the very end for easy access